### ISSUE TITLE
Showing Wrong Input When Type

### ISSUE THREAD

**Issue by MohitBansal321** [didoesdigital/typey-type#142]
### The Problem
When I play the Kaoes game and I have to write -T
In the input section, the hyphen is displayed in the wrong format

### Approach 
We have to remove the underline of character when we type anything

### Screenshot
![image](https://github.com/didoesdigital/typey-type/assets/78220157/a0bc1406-bb31-476f-b6bd-e7730c2e580e)

### Source 
https://didoesdigital.com/typey-type/games/KAOES

**_I will try it if you think this change is significant_**

**Comment by didoesdigital** [comment]
Thanks for this. It's kind of weird seeing an hyphen with an underline!

This textarea is currently using the same styling as other textareas. In lessons the underline is useful for showing where spaces have been typed, but this textarea ignores spaces so it doesn't need the underline. So yeah, you can try remove the underline from here but keep it in lessons.

**Comment by MohitBansal321** [comment]
![image](https://github.com/didoesdigital/typey-type/assets/78220157/5ab51f83-f496-4838-a4ad-d48d618ca515)
Hey @didoesdigital 
Can you help me when I pull the latest code it shows some changes in typed-type-data/

**Comment by didoesdigital** [comment]
You can run the git submodule update command: https://github.com/didoesdigital/typey-type#updates

**Comment by MohitBansal321** [comment]
Thanks for your patience. I was busy with my internship.
Now I can work on it

**Comment by didoesdigital** [comment]
@MohitBansal321 Yes, both `"chromatic"` and `"@types/clipboard"` packages were updated recently. You may need to `git pull`, `git submodule update`, and run `yarn install` to install the latest packages before continuing development with `yarn start`

**Comment by MohitBansal321** [comment]
There are any platform from which we can communicate for this project easily  (discord,slack etc)

**Comment by didoesdigital** [comment]
There's the [Plover](http://www.openstenoproject.org/plover/) Discord `#education-dev` channel

**Comment by didoesdigital** [comment]
Addressed in https://github.com/didoesdigital/typey-type/commit/a5746cc2f6ee5cc43ee8f6c47b1ce66212d65e2c#diff-153aaf98ee92740912789ecbf96e08d74540678554cfba6779c395679293b0fa
